src/System Application/App/Advanced Settings/README.md

```diff
@@ -1,28 +1,3 @@
 The modules exposes advanced settings page and related integration events.
  
 
-# Public Objects
```

Comment:
Do we have a way of populating the reference docs on https://microsoft.github.io/BCApps/ with documentation from the release branches?